frontend/src/Initialize.tsx

```diff
@@ -51,7 +51,7 @@ const DifficultySettings = (props: IDifficultyProps): JSX.Element => {
               Non-trump cards, except the highest
             </option>
             <option value="PointCardNotAllowed">
-              Non-trump, non-point cards
+              Non-trump, non-point cards, unless the highest
```

Comment:
Do you mind making this a bit more clear??
e.g.

`Non-trump, non-point cards (except K when playing A)`